iteration/iteration_test.go

```diff
@@ -346,13 +340,12 @@ func (s *TestIterationRepository) TestLoadChildren() {
 	space, err := repoSpace.Create(context.Background(), &newSpace)
 	assert.Nil(t, err)
 
-	userActive := false
 	repo := iteration.NewIterationRepository(s.DB)
 	level0IterationName := "Top level iteration"
 	i1 := iteration.Iteration{
 		Name:       level0IterationName,
 		SpaceID:    space.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c